chromadb/utils/AutoModelForSentenceEmbedding.py

```diff
@@ -0,0 +1,37 @@
+from torch import nn
```

Comment:
It's odd to have this file as a top level util, since it's only used in the one embedding function. I think this is a hint to us that we need to break up the EF file so that they can be separated and their utility / helper functions can be packaged with them.